Formula/frotz.rb

```diff
@@ -32,11 +31,13 @@ def install
     system "make", "all", *args
     ENV.deparallelize # install has race condition
     system "make", "install_all", *args
+    prefix.install "src/test/etude/etude.z5"
```

Comment:
It's better to download this from the repo in the test with a resource block than to install it for all users just for the test.

Reply:
It sure would have been nice if the documentation had said anything about this! I've filed a PR on the Formula Cookbook to document how this is done. https://github.com/Homebrew/brew/pull/7548

Reply:
And so the documentation gets a little better each time. Tanks for that!